scriptlets/clean_machine

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
This is supposed to run on the DUT, right? I think this is perhaps something that needs to be explicit in the "docstring". Also, if it's supposed to run on the device then it needs to be one of the scriptlets that is copied over to the device in the [installer script](https://github.com/canonical/hwcert-jenkins-tools/blob/fca2f9f2269dcd0f6120feaf1c75c157854accc8/install_tools.sh#L38).

Just to note (and I am really not sure of the answer myself): so far we have been running this sort of thing from the agent using individual `_run` commands. This shifts the approach to running the commands/script directly on the device. I am not sure which is best but it does need to be weighed and considered.

Reply:
I think this is a bit easier to read as is, as basically every  command in this would use `_run` and I can see only downsides to it. The only thing we can get out of `_run`ing every command in this via individual `_run` is network latency, or am I missing something?

Reply:
Also, I expected the installer script to copy everything in the scriplet dir with a glob, any reason we are maintaining an explicit list over there?

Reply:
Fixed beside 1